osu.Game.Rulesets.tau/UI/Cursor/tauCursor.cs

```diff
@@ -41,7 +41,7 @@ private void load(IBindable<WorkingBeatmap> beatmap)
             this.beatmap.BindTo(beatmap);
         }
 
-        public bool CheckForValidation(DrawabletauHitObject h) => h.ScreenSpaceDrawQuad.AABBFloat.IntersectsWith(defaultCursor.HitReceptor.ScreenSpaceDrawQuad.AABBFloat);
+        public bool CheckForValidation(DrawabletauHitObject h) => (h as DrawabletauHitObject).IntersectArea.ScreenSpaceDrawQuad.AABBFloat.IntersectsWith(defaultCursor.HitReceptor.ScreenSpaceDrawQuad.AABBFloat);

```

Comment:
`h` is already defined as `DrawabletauHitObject`, no need to cast it.

Reply:
Oh my bad, I thought it was a generic DrawableHitObject.